core/src/main/scala/org/http4s/MimeDB.scala

```diff
@@ -0,0 +1,7021 @@
+// Autogenerated file, don't edit
+package org.http4s
+
+private[http4s] object MimeDB {
```

Comment:
Note I've made `MimeDB` private. It is the best for MiMa but could make like harder for clients that want to refer to the mime types. WDYT?

Reply:
I guess I'm still finding it a little bit weird that these aren't just mixed into the `MediaType` companion.  What's the disadvantage to that?

Reply:
do you mean `object MediaType extends MimeDB`?
That would be possible. I'll check the changes needed for that

Reply:
Yes, that's exactly what I mean.

Maybe this is a bad idea, because it will lead to thousands of members in a common type.  It might be miserable for people doing autocompletes.  But it's where I'd look for them.